cmd/openshift/operator/kodata/tekton-addon/1.4.0/optional/quickstarts/install-associate-pipeline-quickstart.yaml

```diff
@@ -53,26 +53,28 @@ spec:
       success: You just located the associated Pipeline! Next, we'll start and explore
         your Pipeline.
       failed: Check your work to locate the associated pipeline.
-  - title: Starting your pipeline
+  - title: Exploring your pipeline run
     description: |-
-      ### You’ve just explored the topology of your application and seen it’s related resources.  Now let’s start your pipeline.
+      ### You’ve just explored the topology of your application and seen it’s related resources.  Now let’s explore your pipeline run.
       Notice the Pipeline Runs section of the Side Panel
       1. The first row shows the Pipeline associated with the application and an action button to **Start Last Run**.
```

Comment:
@karthikjeeyar let's just use:
1. The first row shows the Pipeline associated with the application.